sklearn/svm/tests/test_sparse.py

```diff
@@ -24,11 +24,7 @@
     [
         [0, 0, 0],
         [1, 1, 1],
-        [
-            2,
-            0,
-            0,
-        ],
+        [2, 0, 0],
```

Comment:
This is a simple example of how black changes formatting depending on the trailing comma.